example/lib/main.dart

```diff
@@ -17,39 +25,32 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
-    // Can't show a dialog in initState, delaying initialization
-    WidgetsBinding.instance!.addPostFrameCallback((_) => initPlugin());
+    initPlugin();
   }
 
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlugin() async {
-    // Platform messages may fail, so we use a try/catch PlatformException.
-    try {
```

Comment:
PlatformException can not occur, because we are not using any `result(FlutterError(...))` statement in platform code. System exceptions crashes the app, so flutter framework can not handle them to return PlatformException.